wiz-admission-controller/templates/deploymentmanager.yaml

```diff
@@ -0,0 +1,210 @@
+      {{- if .Values.hostNetwork }}
+      hostNetwork: true
+      {{- end }}
+      terminationGracePeriodSeconds: {{ .Values.global.podTerminationGracePeriodSeconds }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- if hasKey .Values.global "lowPrivilegeSecurityPolicy" }}
+            {{- toYaml .Values.global.lowPrivilegeSecurityPolicy | nindent 12 }}
+            {{- else }}
+            {{- toYaml .Values.securityContext | nindent 12 }}
+            {{- end }}
+          image: "{{ coalesce .Values.global.image.registry .Values.image.registry }}/{{ coalesce .Values.global.image.repository .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          readinessProbe:
+            httpGet:
+              path: /ready
+              port: {{ .Values.healthPort }}
+              scheme: HTTPS
+            {{- toYaml .Values.probes.readinessProbe | nindent 12 }}
+          livenessProbe:
+            httpGet:
+              path: /live
+              port: {{ .Values.healthPort }}
+              scheme: HTTPS
+            {{- toYaml .Values.probes.livenessProbe | nindent 12 }}
+          startupProbe:
+            httpGet:
+              path: /ready
+              port: {{ .Values.healthPort }}
+              scheme: HTTPS
+            {{- toYaml .Values.probes.startupProbe | nindent 12 }}
+
+          command:
+          - "/usr/bin/wiz-admission-controller"
+          - "manager"
+          - "--readiness-port={{ .Values.healthPort }}"
+          - "--status-port={{ .Values.statusPort }}"
+          # Server flags
+          - "--tls-private-key-file=/var/server-certs/tls.key"
+          - "--tls-cert-file=/var/server-certs/tls.crt"
+          # Auto update flags
+          - "--auto-update-enabled={{ .Values.wizManager.autoUpdate.enabled }}"
+          - "--release-namespace={{ .Release.Namespace }}"
+          - {{ include "autoUpdate.deployments" . | trim | quote }}
+          # Cluster identification flags
+          - "--cluster-external-id={{ coalesce .Values.global.clusterExternalId .Values.webhook.clusterExternalId }}"
+          {{- with (coalesce .Values.global.subscriptionExternalId .Values.webhook.subscriptionExternalId) }}
+          - --subscription-external-id
+          - {{ . | quote }}
+          {{- end }}
+          {{- with (coalesce .Values.global.clusterTags .Values.webhook.clusterTags) }}
+          - --cluster-tags
+          - {{ . | toJson | quote }}
+          {{- end }}
+          {{- with (coalesce .Values.global.subscriptionTags .Values.webhook.subscriptionTags) }}
+          - --subscription-tags
+          - {{ . | toJson | quote }}
+          {{- end }}
+          env:
+          - name: WIZ_UPLOAD_BACKEND_STORAGE # TODO: remove once this is removed from the code
+            value: "true"
+          {{- if not .Values.wizApiToken.usePodCustomEnvironmentVariablesFile }}
```

Comment:
You don't have to do it now, but we should move all the "common" flags into helpers file to avoid having three separate copies of it.

Reply:
You're right, I will do it on another PR.
It can be nice to have the entire spec as a common that can be used for all pods. I will see what I can do